justfile

```
@@ -40,13 +40,13 @@ rust-fmt:
     #!/usr/bin/env bash
     set -eux -o pipefail
     cargo +"$RUST_NIGHTLY" fmt --all -- {{ if env_var_or_default("CI", "false") == "true" { "--check" } else { "" } }};
-    cargo sort --grouped --workspace {{ if env_var_or_default("CI", "false") == "true" { "--check --check-format" } else { "" } }}
+    cargo sort --grouped --workspace {{ if env_var_or_default("CI", "false") == "true" { "--check" } else { "" } }}
```

:wrench: this works fine on my machine as well as on the ci. this was introduced with the last toolchain update. just to be sure, have you updated your local `cargo sort` to `1.0.9` to match the ci?
```suggestion
    cargo sort --grouped --workspace {{ if env_var_or_default("CI", "false") == "true" { "--check --check-format" } else { "" } }}
```

---

:facepalm: